frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -97,6 +97,7 @@ class EntityMenu extends Component {
                 }
 
                 const key = item.key ?? item.title;
+                const itemId = `entity-menu-item-${encodeURIComponent(key)}`;
```

Comment:
an id used to pass to an element being targeted by `aria-labelledby`, so we could use `*ByRole("listitem", { name: title })`